tests/components/datadog/test_init.py

```diff
@@ -102,23 +89,25 @@ def test_logbook_entry(self, mock_connection, mock_client):
         }
         handler_method(mock.MagicMock(data=event))
 
-        assert mock_client.event.call_count == 1
-        assert mock_client.event.call_args == mock.call(
+        assert mock_statsd.event.call_count == 1
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6356951)